src/components/ValidatedInput/ValidCreatableSelect.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ *
+ *  * Copyright (c) 2021 AccelByte Inc. All Rights Reserved.
+ *  * This is licensed software from AccelByte Inc, for limitations
+ *  * and restrictions contact your company contract manager.
+ *
+ */
+
+import * as React from "react";
+import { default as classNames } from "classnames";
+import { FieldErrorMessage, FieldHelperText, FieldLabel } from "../Form/utility";
+import { CreatableSelect, ValidSelectProps } from "..";
+
+export const ValidCreatableSelect = ({
+  label,
+  placeholder,
+  options = [],
+  name,
+  value,
+  onChange,
+  errMessage,
+  optionalLabel,
+  isInvalid,
+  isDisabled = false,
+  isRequired = true,
+  isClearable,
+  tooltip,
+  selectDataQa,
+  isMulti = true,
```

Comment:
why isMulti is true by default while in CreatableSelect it's false?

Reply:
this is not intended, will update